Website/Providers/DataProviders/SqlDataProvider/08.00.05.01.SqlDataProvider

```diff
@@ -124,6 +124,383 @@ BEGIN
+	DROP PROCEDURE {databaseOwner}[{objectQualifier}UpdateFile]
+GO
+
+CREATE PROCEDURE {databaseOwner}[{objectQualifier}UpdateFile]
+    @FileId                 Int,
+    @VersionGuid            UniqueIdentifier,
+    @FileName               nVarChar(246),
+    @Extension              nVarChar(100),
+    @Size                   BigInt,
+    @Width                  Int,
+    @Height                 Int,
+    @ContentType            nVarChar(200),
+    @FolderID               Int,
+    @LastModifiedByUserID   Int,
+    @Hash                   VarChar(40),
+    @LastModificationTime   DateTime,
+    @Title                  nVarChar(256),
+    @Description            nVarChar(500),
+    @EnablePublishPeriod    Bit,
+    @StartDate              DateTime,
+    @EndDate                DateTime,
+    @ContentItemID          Int
+AS
+BEGIN
+    SET NOCOUNT OFF;
+    DECLARE @PortalID int;
+    SELECT @PortalID = PortalID FROM {databaseOwner}[{objectQualifier}Folders] WHERE FolderID = @FolderID;
+
+    UPDATE {databaseOwner}[{objectQualifier}Files]
+     SET   FileName             = @FileName,
+           VersionGuid          = @VersionGuid,
+           Extension            = @Extension,
+           Size                 = @Size,
+           Width                = @Width,
+           Height               = @Height,
+           ContentType          = @ContentType,
+           FolderID             = @FolderID,
+           PortalID             = @PortalID,
+           LastModifiedByUserID = @LastModifiedByUserID,
+           LastModifiedOnDate   = GetDate(),
+           SHA1Hash             = @Hash,
+           LastModificationTime = @LastModificationTime,
+           Title                = @Title,
+           Description          = @Description,
+           EnablePublishPeriod  = @EnablePublishPeriod,
+           StartDate            = @StartDate,
+           EndDate              = @EndDate,
+           ContentItemID        = @ContentItemID
+     WHERE FileId = @FileId;
+    
+	-- update the parent folders "updated date" to latest date:
+	IF(@@ROWCOUNT>0 AND @FolderID IS NOT Null)
+		EXEC {databaseOwner}[{objectQualifier}UpdateFolderModifiedOnToCurrentDate] @FolderID;
+
+END; -- Procedure
+GO
+
+If Exists (select * from sys.procedures WHERE object_id = object_id(N'{databaseOwner}[{objectQualifier}UpdateFileContent]'))
+	DROP PROCEDURE {databaseOwner}[{objectQualifier}UpdateFileContent]
+GO
+
+CREATE PROCEDURE {databaseOwner}[{objectQualifier}UpdateFileContent]
+	@FileId      int,
+	@Content     VarBinary(Max)
+AS
+BEGIN
+	UPDATE {databaseOwner}[{objectQualifier}Files]
+	 SET   [Content] = @Content
+	 WHERE [FileId]  = @FileId;
+END; -- Procedure
+GO
+
+/* /DNN-8876 */
+
 /************************************************************/
 /*****              SqlDataProvider                     *****/
 /************************************************************/
```

Comment:
As we are on release 9.2.0, this needs to go into ``09.02.00.SqlDataProvider`` file now.

Reply:
You are correct, but I cannot re-submit on every DNN release, just because you are ignoring community PRs for years.

Reply:
Hi Sebastian - We were behind on PRs, but now we are actively following up..